.circleci/config/jobs/build-go-dev.yml

```diff
@@ -0,0 +1,21 @@
+executor: go
+steps:
+  - checkout
+  - restore_go_cache
+  - attach_workspace:
+      at: .
+  - run:
+      name: Build dev binary
+      command: |
+        set -eux -o pipefail
+
+        # Move dev UI assets to expected location
```

Comment:
building a dev binary won't do anything with the UI assets so I'm not sure why these commands are here

Reply:
Hmm, this is just a mechanical moving around of things, so I didn't really examine what they were doing. I would rather leave this here and submit a follow-up PR to remove it as that will make merging across easier.